sat/data/schema/bootprep_schema.yaml

```diff
@@ -34,6 +51,38 @@ description: >
 type: object
 additionalProperties: false
 properties:
+  schema_version:
+    description: >
+      The version of the bootprep input file schema used by this input file. The
+      version of the schema should be a SemVer-like version number with three
+      components, major.minor.patch. This will be populated with the correct
+      value in input files generated by 'sat bootprep generate-example'.
+
+      This version will be checked against the current schema version understood
+      by the 'sat bootprep' commmand.
+
+      Compatibility is defined as follows. Let us refer to the schemaVersion
```

Comment:
nit: should `schemaVersion` be camelCase here?

Reply:
Nope, I think I originally had it as camelCase due to some confusion in my head between the camelCase properties used to define a schema using JSON Schema (i.e. properties in the JSON Schema metaschema). Then I realized this is just a new property in the bootprep input file schema, and those should be lowercase with underscores separating words. I'll fix this.